### DISCUSSION TITLE
Do we have any GitHub Club program for schools/university campus?

### DISCUSSION THREAD

**Discussion by SachinMakwana** [community/community#29782] [Other Feature Feedback, Questions, & Ideas] [answered]
Like, Google Developer Students Clubs, do we have such kind of club program available on GitHub for schools/college/university campuses?

**Comment by melvincwng** [comment, marked as answer]
@SachinMakwana Hi Sachin, yes we do!

You can consider applying for the GitHub Campus Expert program over here: https://education.github.com/experts

Take note, applications to this program is very competitive, so if you don't get in this year, feel free to reapply next year!

All the best, hope this answers your question 😊

**Reply by SachinMakwana** [reply]
Thanks @melvincwng
